docs/reference/setup/important-settings/discovery-settings.asciidoc

```diff
@@ -68,7 +69,8 @@ cluster.initial_master_nodes:
     `cluster.initial_master_nodes`.
 <4> Initial master nodes can also be identified by their IP address.
 <5> If multiple master nodes share an IP address then the transport port must
-    be used to distinguish between them.
+    be used to distinguish between them. Nodes should have unique names so you
+    cannot distinguish named nodes by port.
```

Comment:
I think we should just scratch the IP:PORT in this "getting started" section.

Reply:
Good idea, yes.